var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -44,6 +44,23 @@ class Cp2k(MakefilePackage):
             description=('Enable planewave electronic structure'
                          ' calculations via SIRIUS'))
 
+    # override cuda_arch from CudaPackage since we only support one arch
+    # at a time and only specific ones for which we have parameter files
+    # for optimal kernels
+    variant('cuda_arch',
+            description='CUDA architecture',
+            default='none',
+            values=('none', '35', '37', '60'),
+            multi=False)
+    variant('cuda_arch_35_k20x', default=False,
+            description=('CP2K (resp. DBCSR) has specific parameter sets for'
+                         'different GPU models. Enable this when building with'
+                         'cuda_arch=35 for a K20x instead of a K40'))
```

Comment:
You need spaces at the end of these strings. Otherwise you'llget a sentencewhere several wordsare combined.

Reply:
thanks for catching that, fixed in latest force push